sites/fast-website/src/public/footer.ejs

```diff
@@ -1,11 +1,11 @@
 <footer class="footer">
     <site-navigation class="in-footer">
         <site-navigation-item href="https://github.com/microsoft/fast/blob/master/LICENSE">License</site-navigation-item>
-        <site-navigation-item href="https://privacy.microsoft.com/en-US/privacystatement">Privacy & Cookies</site-navigation-item>
-        <site-navigation-item href="https://www.microsoft.com/en-us/legal/intellectualproperty/copyright/default.aspx">Terms of Use</site-navigation-item>
-        <site-navigation-item href="https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general">Trademarks</site-navigation-item>
+        <site-navigation-item href="https://go.microsoft.com/fwlink/?LinkId=521839">Privacy & cookies</site-navigation-item>
+        <site-navigation-item href="https://go.microsoft.com/fwlink/?LinkID=206977">Terms of use</site-navigation-item>
+        <site-navigation-item href="https://www.microsoft.com/trademarks">Trademarks</site-navigation-item>
```

Comment:
Is trademarks not under the same requirement?

Reply:
no

Reply:
Interesting...👍